### DISCUSSION TITLE
I'm having issues with Samsung S22 photos taken at high res

### DISCUSSION THREAD

**Discussion by kbekus** [photoprism/photoprism#2754] [Q&A] [answered]
hey all, does anyone else have issues with photos taken on a Samsung S22 Ultra? Both Photoprism and Nextcloud refuse to load them for me. Lower res photos taken with the front facing camera work fine.

Nothing to do with permissions and I've changed the max file size to -1.  Any clues as to what I should do next please?

Using exiftool inside the container shows valid data. The only thing I see after enabling trace logging is the error "found no jpeg for photo uid....". Searching for the UID returns no hits and looking in the folder in originals comes up with no images.

I can email some samples to samples@photoprism if it helps? They're between 15 and 20MB.

**Comment by graciousgrey** [comment]
Yes, feel free to mail us some sample files. This way we can check if it is a general issue or a problem with your specific setup.

**Comment by kbekus** [comment]
OK thanks, 2 samples sent in

**Comment by graciousgrey** [comment, marked as answer]
Your files have a resolution of 108 MB.  The current maximum setting for the community edition is 100MB.
You can either wait for the next release which will increase the resolution limit for the community edition to 150MB or [sign up as a sponsor](https://photoprism.app/editions) to set a custom resolution limit.

https://docs.photoprism.app/release-notes/

https://github.com/photoprism/photoprism/discussions/2677

https://docs.photoprism.app/getting-started/config-options/#storage

**Comment by kbekus** [comment]
Thanks very much for your help Theresa

**Reply by graciousgrey** [reply]
You are welcome :)
